docs/core/tools/extensibility.md

```diff
@@ -67,34 +74,34 @@ API, here is a console application's project file that uses that tool:
 
-Since tools are portable applications, the user consuming the tool has to have the version of the .NET Core libraries 
+Since tools are portable applications, the user consuming the tool must have the version of the .NET Core libraries 
 that the tool was built against in order to run the tool. Any other dependency that the tool uses and that is not 
 contained within the .NET Core libraries is restored and placed in the NuGet cache. The entire tool is, therefore, run 
 using the assemblies from the .NET Core libraries as well as assemblies from the NuGet cache. 
 
-These kind of tools have a dependency graph that is completely separate from the dependency graph of the project that 
-uses them. The restore process will first restore the project's dependencies, and will then restore each of the tools and 
+These kinds of tools have a dependency graph that is completely separate from the dependency graph of the project that 
+uses them. The restore process first restores the project's dependencies and then restores each of the tools and 
 their dependencies. 
 
-You can find richer examples and different combinations of this in the [.NET Core CLI repo](https://github.com/dotnet/cli/tree/rel/1.0.0-preview2/TestAssets/TestProjects). 
-You can also see the [implementation of tools used](https://github.com/dotnet/cli/tree/rel/1.0.0-preview2/TestAssets/TestPackages) in the same repo. 
+You can find richer examples and different combinations of this in the [.NET Core CLI repo](https://github.com/dotnet/cli/tree/rel/1.0.1/TestAssets/TestProjects). 
+You can also see the [implementation of tools used](https://github.com/dotnet/cli/tree/rel/1.0.1/TestAssets/TestPackages) in the same repo. 
 
 ### Custom targets
-NuGet has had the capability to package custom MSBuild target and props files for a while now and you can find the official documentation on this on the [NuGet documentation site](https://docs.microsoft.com/nuget/create-packages/creating-a-package#including-msbuild-props-and-targets-in-a-package). With the move in the CLI to using MSBuild, the same mechanism of extensibility applies to .NET Core projects. You would use this type of extensiblity when you want to extend the build process or when you want to access any of the artifacts in the build process, such as generated files or inspect the configuration under which the build is invoked etc. 
+NuGet has had the capability to [package custom MSBuild targets and props files](https://docs.microsoft.com/nuget/create-packages/creating-a-package#including-msbuild-props-and-targets-in-a-package) for a while. With the move of the .NET Core CLI tools to use MSBuild, the same mechanism of extensibility now applies to .NET Core projects. You would use this type of extensibility when you want to extend the build process, or when you want to access any of the artifacts in the build process, such as generated files, or you want to inspect the configuration under which the build is invoked, etc.
```

Comment:
I'd rewrite the first sentence to simply mention the capability, not its historical background.